### PR TITLE
Make installation of DKIM related things optional

### PR DESCRIPTION
**Pull request by Carbenium**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -99,6 +99,8 @@ mailserver_replication_user_home: "{{ mailserver_mailbox_home }}"
 # here
 mailserver_mailbox_home: /var/mail
 
+mailserver_have_dkim: False
```
Comment (uubk) [inline defaults/main.yml]: Default to true for backwards compatibility?

Comment (Carbenium) [reply defaults/main.yml]: Sure thing ;)

Comment (Carbenium) [reply defaults/main.yml]: Done.

**Review by uubk — APPROVED** [review]
LGTM!

**Pull request merged**